[WIP] saas_demo: make demo link

trojikman opened the pull request.

Commit:
:book: renaming to Quick Auth (Master)

i18n-bot committed:
:alien: Update auth_quick_master.pot

Done by forked MQT: https://github.com/it-projects-llc/maintainer-quality-tools/

i18n-bot committed:
:alien: Update saas.pot

Done by forked MQT: https://github.com/it-projects-llc/maintainer-quality-tools/

i18n-bot committed:
:alien: Update saas_public.pot

Done by forked MQT: https://github.com/it-projects-llc/maintainer-quality-tools/

Commit:
:heart_eyes: add possibility to duplicate template's record

Commit:
:sos: add funding button

Commit:
:memo: save future code sketches

kaadevelop reviewed (commented):
**1 installable** module is updated:
```
└─ saas_demo/
   ├─ __manifest__.py
   ├─ controllers/
   |  └─ main.py
   ├─ models/
   |  ├─ saas_demo.py
   |  └─ saas_operator.py
   ├─ security/
   |  └─ ir.model.access.csv
   └─ views/
      └─ saas_demo_view.xml
```

Not installable modules remain unchanged.
```
```

No new features in *doc/changelog.rst* files of installable modules

> sent by [:v: Review Bot](https://odoo-devops.readthedocs.io/en/latest/git/github-review-bot.html)

Commit:
:fire: no need to check template creating without restarting odoo

Commit:
:bath: split the logic for the analysis and installation directory

Commit:
:fire: no need to setup base saas env

Commit:
:bath: change method name

Commit:
:bath: fix copied methods

Commit:
:shield: notify to telegram on issue updates

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:ambulance: update master_url in the created builds when changing the direct_url field in the operator

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:shield:

Commit:
:bath: make method private, add description

Commit:
:bath: use set_param

Commit:
:fire: put tests on saas_demo in a separate module

Commit:
:bath: fix ode comment

Co-Authored-By: Ivan Yelizariev <yelizariev@it-projects.info>

Commit:
:bath: remove duplicate code

Commit:
:memo: add TODO

Commit:
:rainbow: make prepare_template_next easy to patch

Commit:
:bath: rename and isolate checking test method

Commit:
:memo: fix roadmap item

Commit:
:sparkles: don't use RPC on preparing template database (#61)

i18n-bot committed:
:alien: Update saas.pot

Done by forked MQT: https://github.com/it-projects-llc/maintainer-quality-tools/

Commit:
:bath: expression did not execute as needed
